astro-ts-mode.el

```diff
@@ -150,7 +148,13 @@
   (unless (treesit-ready-p 'tsx)
     (error "Tree-sitter grammar for Typescript/TSX isn't available"))
 
-  (treesit-parser-create 'astro)
+  (require 'typescript-ts-mode)
```

Comment:
Are these requires necessary? They're already at the top of the file, they shouldn't be required again.